examples/step-28/step-28.cc

```diff
@@ -1664,8 +1663,8 @@ namespace Step28
               }
 
             ++iteration;
-          }
-        while ((error > parameters.convergence_tolerance) && (iteration < 500));
+        } while ((error > parameters.convergence_tolerance) &&
```

Comment:
This seems wrong -- the closing brace no longer lines up vertically with the opening brace...